addons/mail/models/mail_thread.py

```diff
@@ -569,6 +579,8 @@ def _message_track(self, field_names, initial_values):
             except MissingError:
                 continue
 
+        # find content to log as body
+        bodies = self.env.cr.precommit.data.pop(f'mail.tracking.message.{self._name}', {})
```

Comment:
So this basically looks like "context usage but without the context", right?
A bit strange and confusing.

Reply:
This is done like the tracking actually. It accumulates data and generate a message in the end. In addition to values, we may now specificy a body.